test/extension/handlers/topologymutation/handler.go

```diff
@@ -86,3 +248,23 @@ func (h *Handler) ValidateTopology(ctx context.Context, _ *runtimehooksv1.Valida
 
 	resp.Status = runtimehooksv1.ResponseStatusSuccess
 }
+
+// getStringValueForVariable Get the variable value as JSON string.
```

Comment:
Can we add a TODO (to the GetVariableValue func) that for the final exported util package we need a better API?

I'm think about something where you can just get a variable of a specific type like with the unstructured API, e.g.
`variables.String`, `Variables.Bool`, ...